src/components/VolumeRendering.vue

```diff
@@ -118,10 +118,8 @@ export default {
   },
 
   computed: {
-    ...mapState({
-      baseImage: 'selectedBaseImage',
-      colorBy: (state) => state.visualization.colorBy,
-    }),
+    ...mapState({ baseImage: 'selectedBaseImage' }),
+    ...mapState('visualization', ['colorBy']),
     ...mapGetters(['baseImagePipeline', 'baseImageColorPreset']),
```

Comment:
These are part of the visualization namespace (check out the getters in `src/store/visualization.js`).